web/src/app/util/OtherActions.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import React, {useState} from 'react'
```

Comment:
I'm noticing some linting issues here, running `yarn lint --fix` should fix the issue. This will cause `make check` to fail until then